packages/app/README.md

```diff
@@ -4,11 +4,12 @@ This is the front-end for the Cypress App.
 
 ## Development
 
-1. `yarn dev` (inside of `packages/app`)
-2. It will open launchpad
-3. Select Component or E2E Testing
-3. Open chrome (or another browser)
-4. It will show the new Vite powered app 
+1. `yarn watch` (inside of root)
+1. `yarn cypress:open` (inside of `packages/app`)
```

Comment:
Could `yarn watch` and `yarn cypress:open` be combined into one command for convenience or are two windows required?

Reply:
@elylucas that's what `yarn dev` does

Reply:
> maybe add another example to show opening up a specific project like yarn cypress:open --project /path/to/project

Good point. This is specifically called out to run this way because it's the `app` documentation, because you can't start with `yarn cypress:open --project` for the app. Could call it out elsewhere

Reply:
I meant specifically for `--project projects/app` it doesn't work